tests/api/utils_post_data.py

```diff
@@ -40,7 +40,8 @@
         "privacy": "public",
         "start_date": "2016-06-29T19:30:00",
         "timezone": "UTC"
-    }
+    },
+    "identifier": "23c0644e"
```

Comment:
@rafalkowalski I don't think identifier will be a part of Event POST API input payload as it is machine set.